x/gamm/pool-models/stableswap/amm.go

```diff
@@ -287,7 +292,7 @@ func solveCFMMBinarySearchMulti(xReserve, yReserve, wSumSquares, yIn osmomath.Bi
 	}
 
 	xOut := xReserve.Sub(xEst)
-	if xOut.GTE(xReserve) {
+	if xOut.Abs().GTE(xReserve) {
```

Comment:
Lets add a comment as to why its abs?

Reply:
Lets add a comment that this ensures that:
* We can't swap more than the pool supply out
* A swap won't double the input asset supply